src/components/relativeLink/index.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { withRouter } from 'react-router';
+import buttonStyle from 'react-toolbox/lib/button/theme.css';
+import offlineStyle from '../offlineWrapper/offlineWrapper.css';
+
+const RelativeLink = ({
+  location, to, children, className, raised, neutral, primary, flat, disableWhenOffline,
+}) => {
+  let style = '';
+  if (raised !== undefined) style += `${buttonStyle.raised} `;
+  if (neutral !== undefined) style += `${buttonStyle.neutral} `;
+  if (flat !== undefined) style += `${buttonStyle.flat} `;
+  if (primary !== undefined) style += `${buttonStyle.primary} `;
+  if (disableWhenOffline !== undefined) style += `${offlineStyle.disableWhenOffline} `;
+  if (style !== '') style += ` ${buttonStyle.button}`;
+
```

Comment:
I think that https://github.com/JedWatson/classnames is better solution for dynamic classnames

Reply:
There's a ticket to fix many other issues about this utility.